source/simulator/assembly.cc

```diff
@@ -1192,7 +1274,17 @@ namespace aspect
   template <int dim>
   void Simulator<dim>::assemble_stokes_system ()
   {
-    computing_timer.enter_section ("   Assemble Stokes system");
+    if (!assemble_newton_stokes_system)
+      computing_timer.enter_section ("   Assemble Stokes system");
+    else if (assemble_newton_stokes_matrix)
+      {
+        if (parameters.newton_theta == 0)
+          computing_timer.enter_section ("   Assemble Stokes system picard");
```

Comment:
it is the defect correction form of the Picard iteration. It should be practically the same as the normal Picard iteration, but may be slightly more accurate. I thought stateing `   Assemble defect correction form of Stokes system picard` would be a bit long. But since it should not matter much, I could indeed add it to the normal`assemble stokes system`.